drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -1419,6 +1419,99 @@ Variant RasterizerStorageGLES2::material_get_param(RID p_material, const StringN
 		return material->params[p_param];
 	}
 
+	return material_get_param_default(p_material, p_param);
+}
+
+Variant RasterizerStorageGLES2::material_get_param_default(RID p_material, const StringName &p_param) const {
+	const Material *material = material_owner.get(p_material);
+	ERR_FAIL_COND_V(!material, Variant());
+
+	if (material->shader) {
+		if (material->shader->uniforms.has(p_param)) {
```

Comment:
I was thinking that to avoid repeating this code in GLES2 and GLES3, there could be a function to convert a GLSL value into a Variant in ShaderLanguage?